### DISCUSSION TITLE
Webradio Folder Bookmark Not Working

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1810] [Bug Reports]
@rern 
I have webradio folders which I can bookmark complete with a thumbnail image, but the bookmark shows a blank page when clicked. 

EDIT: I also notice that the same folders display as thumbnails in the 20240113 image.

**Comment by rern** [comment]
Please try the UPDATE.

**Reply by R101G** [reply]
Thanks @rern the bookmark works now, complete with thumbnail. The folder icon in the webradio list does not show the thumbnail.
